crashreporter/src/main/java/com/tealium/crashreporter/Crash.kt

```diff
@@ -0,0 +1,162 @@
+
+final class Crash (Thread thread, Throwable ex) {
+    const val KEY_STACK_CLASS_NAME = "className"
+    const val KEY_STACK_FILENAME = "fileName"
+    const val KEY_STACK_LINE_NUMBER = "lineNumber"
+    const val KEY_STACK_METHOD_NAME = "methodName"
+    const val KEY_THREAD_STATE = "state"
+    const val KEY_THREAD_NUMBER = "threadNumber"
+    const val KEY_THREAD_ID = "threadId"
+    const val KEY_THREAD_PRIORITY = "priority"
+    const val KEY_THREAD_STACK = "stack"
+
+    private val mException: Throwable
```

Comment:
`m...` prefixed not used in [Kotlin properties](https://kotlinlang.org/docs/reference/properties.html)